### PR TITLE
Cypress/fe 1641

### PR DESCRIPTION
**Pull request by marcinstachowicz**
## Automated BDD test specs for components:

Spinner,
SplitButton,
Step sequence / Step sequence Item (in progress)
Switch

### CODE REVIEW

File: cypress/integration/spinner.feature [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      | new         |
+      | success     |
+      | warning     |
+
+  @positive
+  Scenario Outline: I set Spinner component size to <size>
+    When I select size to "<size>"
+    Then Spinner size is set to "<size>"
+    Examples:
+      | size         |
```
Comment (DlgSHi) [inline cypress/integration/spinner.feature]: Can we check also the size in px ?

File: cypress/integration/splitButton.feature [hunk 2]
```diff
@@ -0,0 +1,68 @@
+Feature: Split Button component
+  I want to change Split Button component properties
+
+  Background: Open Split Button component page classic
+    Given I open "Split Button" component page classic
+
+  @positive
+  Scenario Outline: I select as to <as>
+    When I select as to "<as>"
+    Then Button background color is "<color>"
```
Comment (DlgSHi) [inline cypress/integration/splitButton.feature]: if the statement will start from button word, it should start from small letter, but if U want to start statement from Split Button -it should start with Capital character

File: cypress/integration/splitButton.feature [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      | Sample text             |
+      | 1234567890              |
+      | áéíóú¿¡üñ               |
+      | !@#$%^*()_+-=~[];:.,?{} |
+      | ÄÖÜßäöüß                |
+
+  @positive
+  Scenario: I disable Split Button component
+    When I disable SplitButton component
+    Then Button is disabled
```
Comment (DlgSHi) [inline cypress/integration/splitButton.feature]: Split Button or button ->

File: cypress/integration/splitButton.feature [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+  @positive
+  Scenario: I enable Split Button component
+    When I disable SplitButton component
+      And I enable SplitButton component
+    Then Button is enabled
```
Comment (DlgSHi) [inline cypress/integration/splitButton.feature]: Split Button or button ->

File: cypress/integration/splitButton.feature [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+  @positive
+  Scenario Outline: I set text to <text>
+    When I set text to "<text>"
+    Then Button label on preview is "<text>"
```
Comment (DlgSHi) [inline cypress/integration/splitButton.feature]: Split Button or button ->

File: cypress/locators/split-button/index.js [hunk 6]
```diff
@@ -0,0 +1,4 @@
+import { TOGGLE_BUTTON } from './locators';
```
Comment (DlgSHi) [inline cypress/locators/split-button/index.js]: Why locator is called toggle_button, if we have split_button component ?

File: cypress/support/step-definitions/spinner-steps.js [hunk 7]
```diff
@@ -0,0 +1,7 @@
+import { spinner } from '../../locators/spinner';
+
+const SPINNER_CLASS_PREFIX = 'carbon-spinner--';
+
+Then('Spinner {word} is set to {string}', (word, value) => {
```
Comment (DlgSHi) [inline cypress/support/step-definitions/spinner-steps.js]: how `word` parameter is used here ?

File: cypress/locators/split-button/index.js (hunk 6 above)
Comment (marcinstachowicz) [reply cypress/locators/split-button/index.js]: data element is called toggle-button

File: cypress/support/step-definitions/spinner-steps.js (hunk 7 above)
Comment (marcinstachowicz) [reply cypress/support/step-definitions/spinner-steps.js]: for cucumber step to recognize `as` or `size`

File: cypress/integration/spinner.feature (hunk 1 above)
Comment (marcinstachowicz) [reply cypress/integration/spinner.feature]: added

File: cypress/locators/split-button/index.js (hunk 6 above)
Comment (marcinstachowicz) [reply cypress/locators/split-button/index.js]: fixed to splitToggleButton and SPLIT_TOGGLE_BUTTON

File: cypress/integration/splitButton.feature (hunk 2 above)
Comment (marcinstachowicz) [reply cypress/integration/splitButton.feature]: it is capial letter because step is reused

File: cypress/integration/splitButton.feature (hunk 3 above)
Comment (marcinstachowicz) [reply cypress/integration/splitButton.feature]: it is capial letter because step is reused

File: cypress/integration/splitButton.feature (hunk 4 above)
Comment (marcinstachowicz) [reply cypress/integration/splitButton.feature]: it is capial letter because step is reused

File: cypress/integration/splitButton.feature (hunk 5 above)
Comment (marcinstachowicz) [reply cypress/integration/splitButton.feature]: it is capial letter because step is reused

**Pull request merged**